api/v1alpha1/tiflowcluster_types.go

```diff
@@ -455,7 +455,10 @@ type MasterStatus struct {
 
 // MasterMember is Tiflow-master member status
 type MasterMember struct {
-	// member id is actually a uint64, but apimachinery's json only treats numbers as int64/float64
+	Id       string `json:"id,omitempty"`
+	Address  string `json:"address,omitempty"`
+	IsLeader bool   `json:"is_leader,omitempty"`
```

Comment:
Is it possible for this filed to be inaccurate? operator pull status form master at regular intervals , which means there is A delay if leader change happend during that interval

Reply:
Yes, at the moment there seems to be some latency, which needs to be optimized in conjunction with subsequent features of reconcile